templates/docs/rhdm712-authoring-ha.adoc

```diff
@@ -92,29 +92,27 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
 |`SSO_DISABLE_SSL_CERTIFICATE_VALIDATION` | `SSO_DISABLE_SSL_CERTIFICATE_VALIDATION` | RH-SSO Disable SSL Certificate Validation. | false | False
 |`SSO_PRINCIPAL_ATTRIBUTE` | `SSO_PRINCIPAL_ATTRIBUTE` | RH-SSO Principal Attribute to use as user name. | preferred_username | False
 |`AUTH_LDAP_URL` | `AUTH_LDAP_URL` | LDAP endpoint to connect for authentication. For failover, set two or more LDAP endpoints separated by space. | ldap://myldap.example.com:389 | False
+|`AUTH_LDAP_LOGIN_MODULE` | `AUTH_LDAP_LOGIN_MODULE` | LDAP login module flag, adds backward compatibility with the legacy security subsystem on elytron. 'optional' is the only supported value, if set will create a distributed realm with ldap and filesystem realm with the user added using the KIE_ADMIN_USER. | optional | False
```

Comment:
"will create distributed realm" does not appear clear - will the realm be created on the LDAP server, etc? Also LDAP should probably be capitalized?

Reply:
It refers to the Elytron configuration, so these changes are applied only on the JBoss configuration.
let me rephrase it,

```
 LDAP login module flag, adds backward compatibility with the legacy security subsystem on Elytron. 'optional' is the only supported value, if set, it will create a distributed realm on Elytron configuration with LDAP and FileSystem realms with the user added using the KIE_ADMIN_USER.
```

does it sounds better?

Reply:
fixed